rust/xnvme-sys/src/bin/xnvme_rust_example01.rs

```diff
@@ -0,0 +1,12 @@
+use std::process::ExitCode;
+
+fn main() -> ExitCode {
+    println!(
+        "v{}.{}.{}",
+        unsafe { xnvme_sys::xnvme_ver_major() },
+        unsafe { xnvme_sys::xnvme_ver_minor() },
+        unsafe { xnvme_sys::xnvme_ver_patch() },
+    );
+
+    ExitCode::from(0)
+}
```

Comment:
This should probably be an example, see [here](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples)